### ISSUE TITLE
Help wanted/offered

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/covid19#1]
I'm not sure what help we need yet, but probably lots.  

When we get to the production phase there'll likely be laser-cutting, running 3D printers, hole-punching OHP sheets, bagging things up.

However, there's also sourcing materials (once we know what's wanted); "gardening" the issues and documentation here; finding relevant info elsewhere and relaying the important bits here...

I figured it might be useful to have a general help issue that people can volunteer on (feel free to dive in on any specific issue too though!), so that's this one.  Hello :wave:

**Comment by magman2112** [comment]
I can obviously help with keeping Gerald and Sophia in action, as well as contributing to any laser production that may be required. Depending on volumes and getting sufficient volunteers, we may even be able to run shifts of production if required.

On the 3D printing side, I can help here as well, I could also look at my tool collection for any tooling that might help finish any 3D printed items to improve their finish - such as deburring or removing excess material.

I can also help with any assembly, packaging and possibly distribution, as I have a car available for deliveries.

**Comment by magman2112** [comment]
Just thought on. I have at least one 4 hole punch available should it be needed.

I also have a large heavy guillotine, good enough for 100 pages of A4 paper, that might help with some material preparation.

**Comment by huffeec** [comment]
I can use the cutter in Norris Green whenever it's needed. Unfortunately I don't have any suitable materials in ATM.
I also have a vehicle so happy to help with delivery too.

**Comment by amcewen** [comment]
Ace.  Thanks both of you.  

@magman2112 I think the guillotine and hole punch will be handy - the visor production can be super low-tech like that.  And DoES doesn't actually have a four-hole punch - there are something like four two-hole punches...

@huffeec we don't have lots of materials either, so we'll be ordering a bunch in as soon as we know fully what we need.  Then we'll be able to fan out deliveries.

**Comment by ajlennon** [comment]
Amazon seems to have them on next day. Shall I get a couple of four hole punches on order?

**Comment by ajlennon** [comment]
Expensive but this looks like it'd get through a lot of sheets!

https://www.amazon.co.uk/Rapesco-PF4000A1-Heavy-4-Hole-Punch/dp/B000SHW102/ref=sr_1_6?dchild=1&keywords=heavy+duty+four+hole+punch&qid=1585318350&sr=8-6

**Comment by Sean-anotherone** [comment]
Technically I'm required to be working at home, but I can operate and or maintain most of the kit and I have a car.  Certainly able to muck in during silent hours.

**Comment by plastictactics** [comment]
We are setting a laser cutter up in the next week or so, so can act as extra cutting capacity if needed, once the material and design is settled. And I'm generally keen to help on this so will be keeping an eye on things and letting our team know what's going on, in case they want to get involved.
^arthur

**Comment by DoESsean** [comment]
I can deliver when required.

**Comment by drakard** [comment]
Hi just to let you know I can try to help with anything you need. Manual cutting and sticking, tech, admin etc. I just sadly don't own any kit myself. My business is mostly held up by Covid so I have time if you need it.

**Comment by simonholgate** [comment]
I’m freed up to help a bit now too. Happy to run things around in the car or do any other things that need doing.

**Comment by plastictactics** [comment]
For help offered: https://afroleft.typeform.com/to/saIT4z

**Comment by plastictactics** [comment]
For help offered: https://docs.google.com/forms/d/e/1FAIpQLSca6bWcUwvAfzfc2A-vNTZklxa5KxeChzk6gO2pMnESX3pfCg/viewform

For help wanted: Please request it in a relevant issue if there is one or open a new issue with your request.

For an overview of who is involved: read #47

Closed since we do now have a clear picture of the help needed, and to reduce the number of open issues.
